dlhub_sdk/client.py

```diff
@@ -125,13 +136,20 @@ def __init__(self, dlh_authorizer: Optional[GlobusAuthorizer] = None,
         }
 
         login_manager = FuncXLoginManager(authorizers=auth_dict)
-        self._fx_client = FuncXClient(login_manager=login_manager)
+        self._fx_client = FuncXClient(funcx_service_address="https://api.dev.funcx.org/v2", login_manager=login_manager)
 
         self._search_client = globus_sdk.SearchClient(authorizer=search_authorizer,
                                                       transport_params={"http_timeout": http_timeout})
 
+        self._openid_client = globus_sdk.AuthClient(authorizer=openid_authorizer,
+                                                    transport_params={"http_timeout": http_timeout})
+
+        self.userinfo = self._openid_client.oauth2_userinfo()
+        self.sl_authorizer = sl_authorizer
+
         # funcX endpoint to use
-        self.fx_endpoint = '86a47061-f3d9-44f0-90dc-56ddc642c000'
+        # self.fx_endpoint = '86a47061-f3d9-44f0-90dc-56ddc642c000'
+        self.fx_endpoint = '2238617a-8756-4030-a8ab-44ffb1446092'
```

Comment:
This is pointing at the dev tutorial endpoint, not a dlhub-specific endpoint. Just FYI. It is fine for testing, but not ideal for any users.